src/controller/deviceindex/ControllerDisconnectedWindow.cpp

```diff
@@ -139,15 +139,10 @@ void ControllerDisconnectedWindow::DrawElement() {
         return;
     }
 
-    ImGui::OpenPopup("Controller Disconnected");
-    if (ImGui::BeginPopupModal("Controller Disconnected", NULL, ImGuiWindowFlags_AlwaysAutoResize)) {
-        // todo: don't use UINT8_MAX-1 to mean we don't know what controller was disconnected
-        if (mPortIndexOfDisconnectedController == UINT8_MAX - 1) {
-            DrawUnknownOrMultipleControllersDisconnected();
-        } else {
-            DrawKnownControllerDisconnected();
-        }
-        ImGui::EndPopup();
+    if (mPortIndexOfDisconnectedController == UINT8_MAX - 1) {
```

Comment:
i was getting a ton of mismatched open/close popup errors, i assume something changed with modern menu to make it so this can just be simplified - seems to work having it this way now